src/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```

Comment:
we basically have a library module, so maybe we can get rid of the `index.html` and `main.ts`?